src/test/java/lambda/part1/exercise/Exercise3.java

```diff
@@ -0,0 +1,69 @@
+package lambda.part1.exercise;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+import com.google.common.collect.FluentIterable;
+import java.util.Arrays;
+import java.util.List;
+import lambda.data.Person;
+import org.junit.Test;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+public class Exercise3 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, (a, b) -> Integer.compare(a.getAge(), b.getAge()));
+
+        assertArrayEquals(new Person[]{
+            new Person("Иван", "Мельников", 20),
+            new Person("Николай", "Зимов", 30),
+            new Person("Алексей", "Доренко", 40),
+            new Person("Артем", "Зимов", 45)
+        }, persons);
+    }
+
+    @Test
+    public void sortPersonsByLastNameThenFirstNameUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, (a, b) -> {
```

Comment:
It is not Expression Lambda :) Improve it, if you can

Reply:
Sorry, look just to method name, but in ToDo were written that we should usr statement lambda